src/views/profile/profileView.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import NavBar from "../../components/navigation/navBar";
+import PropTypes from "prop-types";
+import { Button } from "reactstrap";
+import decode from "jwt-decode";
+import { AUTHENTICATED } from "../../utils/myHeaders";
+
+const userProfile = props => {
+  let {
+    username,
+    image,
+    bio,
+    first_name,
+    last_name
+  } = props.profile;
+  let isOwner;
+  let loggedInUser = decode(AUTHENTICATED);
+  loggedInUser.username === username ? (isOwner = true) : (isOwner = false);
+
+  let bt = "";
```

Comment:
Use a better naming variable instead of bt to give it more meaning

Reply:
I have changed the name